src/Implementations/AzureTableStorage.cs

```diff
@@ -12,6 +14,7 @@ namespace BaseCap.CloudAbstractions.Implementations
     /// </summary>
     public class AzureTableStorage : ITableStorage
     {
+        private const int MAX_BATCH_SIZE = 100;
```

Comment:
Why is MAX_BATCH_SIZE set to 100? Is this a documented max?

Reply:
Yes. It is documented